win32_deps_build.sh

```diff
@@ -22,6 +22,7 @@ boostSrcDir="${depsSrcDir}/boost_1_85_0"
 boostDir="${depsToolsetDir}/boost"
 zlibDir="${depsToolsetDir}/zlib"
 zlibSrcDir="${depsSrcDir}/zlib"
+zlibTag="v1.3.1"
```

Comment:
Another project fixed it by switching to the master branch (from develop, which is the default).  Master happens to be pointing at v1.3.1 too.  Perhaps it would be better to do that instead of pinning to v1.3.1 for a longer term fix?  I'm asking because this is going to need to be backported to reef and squid.